ros/lib/utils.py

```diff
@@ -31,6 +31,21 @@ def get_or_create(session, model, keys, **kwargs):
     return instance
 
 
+def delete_record(session, model, keys, **kwargs):
```

Comment:
```suggestion
def delete_record(session, model, **kwargs):
```
I don't think this method requires keys argument. can't you simply iterate on `kwargs`

Reply:
updated.